datahub-web-react/src/app/home/HomePageHeader.tsx

```diff
@@ -119,6 +120,13 @@ export const HomePageHeader = () => {
     const user = useGetAuthenticatedUser()?.corpUser;
     const themeConfig = useTheme();
     const appConfig = useAppConfig();
+    const [newSuggestionData, setNewSuggestionData] = useState<GetAutoCompleteMultipleResultsQuery | undefined>();
+
+    useEffect(() => {
```

Comment:
This will only solve on the home page right?

Reply:
Yes.